src/mscorlib/src/System/Reflection/Emit/MethodBuilderInstantiation.cs

```diff
@@ -85,7 +85,7 @@ public override bool ContainsGenericParameters
 
         public override MethodInfo MakeGenericMethod(params Type[] arguments)
         {
-            throw new InvalidOperationException(SR.Arg_NotGenericMethodDefinition);
+            throw new InvalidOperationException(SR.Format(SR.Arg_NotGenericMethodDefinition, this.GetType().ToString()));
```

Comment:
@AtsushiKan Is `this.GetType().ToString()` correct here? I believe that it is going to be the name of the reflection implementation type (e.g. `MethodBuilderInstantiation`) that is not relevant for the error message. I think this should be `this.ToString()`.

Reply:
Yes, good catch.

Reply:
do you revert merge?o i've to create another branch/PR?

Reply:
@MarcoRossignoli Create another branch/PR please. Thanks!